mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule MasterProxy.MixProject do
   def project do
     [
       app: :master_proxy,
-      version: "0.1.3",
+      version: "0.1.4",
```

Comment:
Do you know if this will break apps that upgrade? Perhaps this needs to be a major version upgrade?

Reply:
It doesn't seem to.  The tests pass and it works for my Phoenix 1.5 apps. Happy to modify the version number as you'd like

Reply:
Do you know if it still works with Phoenix 1.4 apps? Like say if I had a 1.4 app running with master_proxy and then one day I ran `mix deps.update master_proxy`.